websites/J/JavaScript/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Vyrek",
+      "id": "538421122920742942"
+    },
+    "url": "developer.mozilla.org",
+    "description": {
+      "es": "Pone en tu presencia lo que estas viendo en los docs JS",
+      "en": "Show the things that are you seeing in the JS docs"
+    },
+    "service": "JavaScript | MDN",
```

Comment:
service name and folder name must be 100% the same, and just "Javascript" should be enough